server/vulnerabilities/nvd/cpe_translations.json

```diff
@@ -119,5 +119,15 @@
       "product": ["flock"],
       "vendor": ["flock"]
     }
+  },
+  {
+    "software": {
+      "bundle_identifier": ["/^com\\.jetbrains\\.intellij/"],
+      "source": ["apps"]
+    },
+    "filter": {
+      "product": ["intellij_idea"],
+      "vendor": ["jetbrains"]
+    }
```

Comment:
Do we still need this with the added changes to `cpeGeneralSearchQuery`?

Reply:
Unfortunately, yes. The bundle is `com.jetbrains.intellij` but the product is `intellij_idea`, not `intellij`.